cpp/ps0005/ps0005_sol.cpp

```diff
@@ -4,50 +4,76 @@ using namespace std;
 
 int main()
 {
-    int a , b ;
-    char o ;
-    int r ;
-    cout<<"Enter the two operands:\n" ;
-    cin>> a>> b ;
-    cout<< "Enter an operator from these (+ - * / %): \n" ;
-    cin >> o ;
-    switch (o)
+    // prompting user for 2 operands and an operator 1.5/2
+    double num1, num2;
+    char op;
+    double res;
+    cout << "Enter the two operands:\n";
+    cin >> num1 >> num2;
+    cout << "Enter an operator from these (+ - * / %): \n";
+    cin >> op;

```

Comment:
- [x] declared variables to store the user input
- [ ] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input